write/writer.go

```diff
@@ -118,25 +125,27 @@ func writeBasicOffsetTable(e *dicomio.Encoder, offsets []uint32) {
 //
 // REQUIRES: Each value in values[] must match the VR of the tag. E.g., if tag
 // is for UL, then each value must be uint32.
-func Element(e *dicomio.Encoder, elem *element.Element) {
+func Element(e *dicomio.Encoder, elem *element.Element, opts Options) {
```

Comment:
This does match the base `dicom` package by adding a simple Options struct. Something to consider though: one way to not break the API and not require folks to have to pass in empty options struct is by using a variadic options parameter, something like this:
```go
func Element (e *dicomio.Encoder, elem *element.Element, opts ...Option)
```

In this way, the caller can provide zero or many `Option`s when constructing an Element, however we will have to think about what the `Option` type could be. Often times the `Option` type is actually a function that is passed an instance of a struct, and that function sets properties on it. We can also simplify it and have `Option` be a struct and we iterate through all the options and essentially reduce them all into a flat `Option`, whereby each successive option's properties would overwrite the prior option's. 

Not to overcomplicate it, but something to think about. I won't hold up the PR on this, since breaking the API here shouldn't be a big deal since we're using versioned go modules and because I'm working on a larger scale refactor / rewrite anyway, and I can/will revisit the API then. Can open up an issue to track.

Reply:
I'm thinking now that it might be a good idea to give the variadic options a shot--if you don't have time, let me know and I can take a look as well.

Thanks!